internal/auditd/sessiontracker/sessiontracker.go

```diff
@@ -235,9 +235,12 @@ func (o *sessionTracker) auditEventWithoutSession(event *aucoalesce.Event, debug
 		}
 	}
 
+	procArgs := event.Process.Args
+
 	u := &user{
-		added:  time.Now(),
-		srcPID: srcPID,
+		added:    time.Now(),
+		srcPID:   srcPID,
+		procArgs: procArgs,
```

Comment:
Per the comment below, these code changes can be removed.

Reply:
Updated as per the review...Thanks @stephen-fox